bgpd/bgp_evpn_vty.c

```diff
@@ -6269,6 +6269,10 @@ DEFUN (bgp_evpn_vrf_rd,
 
 	/* Configure or update the RD. */
 	evpn_configure_vrf_rd(bgp_vrf, &prd, argv[1]->arg);
+
```

Comment:
This fix belongs in evpn_configure_vrf_rd

Add a line that free's the old one before the strdup

Reply:
Done. Thank you for the feedback!

Reply:
this was not done btw.

Reply:
it was sorry.